### ISSUE TITLE
Landing page improvements and refactoring

### ISSUE THREAD

**Issue by orhun** [orhun/rustypaste#52]
A landing page example was added in #51 and it demonstrates how to use a HTML form for uploads. To extend this, we can introduce a new option called `landing_page_file` which will read the contents of a file to serve it as the landing page. The reasoning behind this is that an user might not want to put X lines of HTML code into the configuration file.

```toml
[server]
# landing_page = ""
landing_page_file = "index.html"
landing_page_content_type = "text/html; charset=utf-8"
```

Another idea is to refactor the landing page configuration into a separate section:

```toml
[landing_page]
# text = ""
file = "index.html"
content_type = "text/html; charset=utf-8"
```

I think this would be cleaner but we can go with the first option as the first stage of the implementation. More comments needed.

**Comment by vihu** [comment]
Just a suggestion, it would also be nice to allow a user of the landing page to provide server side auth_token in the input form.

**Comment by orhun** [comment]
This is now implemented in #65 

> Just a suggestion, it would also be nice to allow a user of the landing page to provide server side auth_token in the input form.

Feel free to submit a new issue for tracking!
